### PR TITLE
B2C sample preconfigured to Fabrikam

### PR DESCRIPTION
**Pull request by bgavrilMS**
## Purpose

Preconfigures chapter 1.5 with the Fabrikam B2C demo env

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[x] Other... Settings changes
```

## How to Test
Run chapter 1.5 without making any changes. You should be able to login / logout and view the Claims.

### CODE REVIEW

File: 1-WebApp-OIDC/1-5-B2C/README.md [hunk 1]
```diff
@@ -100,6 +108,9 @@ If your web site needs to be accessed from users using iOS 12, you probably want
 
 ## About The code
 
+#### Where is MSAL?
+This sample does NOT use MSAL - it uses the built-in ASP.NET Core middlerware. MSAL is used for fetching access  for accessing protected APIs (not shown here), as well as ID tokens. For logging-in purposes, it is sufficient to obtain an ID Token, and the middlerware is capable of doing this on its own.
```
Comment (jmprieur) [inline 1-WebApp-OIDC/1-5-B2C/README.md]: Typo: middlerware => middleware (twice)

File: 1-WebApp-OIDC/1-5-B2C/appsettings.json [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-		}
-	},
-	"AllowedHosts": "*"
+  "AzureAdB2C": {
+    "Instance": "https://fabrikamb2c.b2clogin.com",
+    "ClientId": "90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6",
+    "Domain": "fabrikamb2c.onmicrosoft.com",
+    "SignedOutCallbackPath": "/signout/B2C_1_susi",
+    "SignUpSignInPolicyId": "B2C_1_susi",
+    //"CallbackPath": "/signin/B2C_1_sign_up_in"  // defaults to /signin-oidc
```
Comment (jmprieur) [inline 1-WebApp-OIDC/1-5-B2C/appsettings.json]: do we want to remove the line altogether?

File: 1-WebApp-OIDC/1-5-B2C/README.md (hunk 1 above)
Comment (jmprieur) [inline 1-WebApp-OIDC/1-5-B2C/README.md]: This sample does NOT use MSAL as it only signs-in users (it does not call a Web API).

File: 1-WebApp-OIDC/1-5-B2C/appsettings.json (hunk 2 above)
Comment (bgavrilMS) [reply 1-WebApp-OIDC/1-5-B2C/appsettings.json]: It's still mentioned in the readme

**Pull request merged**